tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -331,19 +331,19 @@ def test_getUsers(self):
                                   [field % ('user_3', 'user_3', '', '43200')],
                                   '(UserName, UserDN, Pem, ExpirationTime)')]:
       result = db._update('INSERT INTO %s%s VALUES %s ;' % (table, fields, ', '.join(values)))
-      self.assertTrue(result['OK'], '\n%s' % result.get('Message') or 'Error message is absent.')
+      self.assertTrue(result['OK'], '\n' + (result.get('Message') or 'Error message is absent.'))
```

Comment:
```suggestion
      self.assertTrue(result['OK'], '\n' + result.get('Message', 'Error message is absent.'))
```

https://docs.python.org/2.7/library/stdtypes.html#dict.get

Reply:
already fixed